httpcore5/src/test/java/org/apache/hc/core5/concurrent/TestBasicFuture.java

```diff
@@ -260,41 +260,41 @@ public void cancelled() {
 
     @Test
     void testGetWithTimeout() {
+        final AtomicBoolean isFutureCompleted = new AtomicBoolean(false);
```

Comment:
@arturobernalg Looks good, but there appears to be unrelated change in the change-set. Please move it into a separate commit or a separate PR.

Reply:
@ok2c. fair enough. Changed.